src/Goteo/Controller/Api/ChartsApiController.php

```diff
@@ -73,7 +73,7 @@ protected function getMatcher($matcher, $private = false) {
      * Simple projects info data specially formatted for D3 charts
      * COSTS
      */
-    public function projectCostsAction($id) {
+    public function projectCostsAction($id, Request $request) {
```

Comment:
To let controllers that inherit from this controller use the `Request` parameter, the parent needs to have it too.